client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -111,29 +123,90 @@ export default defineComponent({
       return 'mdi-image-multiple';
     }
 
+    async function preloadCheck(recent: JsonMetaCache) {
+      //Attempts to preload the data to see if there are any isues
+      try {
+        await api.checkDataset(recent.id);
+      } catch (e) {
+        const result = await prompt({
+          title: 'Error Loading Data',
+          text: e,
+          confirm: true,
+          positiveButton: 'Delete',
+          negativeButton: 'Cancel',
+        });
```

Comment:
This is something to note for the future. In the CLI Tools PR, there is a new function to validate datasets. One thing we can do is call that validate function when attempting to open a dataset to check if it's corrupted and notify the user. 

Not something necessarily for this PR, but thought it's worth mentioning.

Reply:
The error here could be a missing metadata file, missing project folder or missing media source folder.  I've updated it to to remove the in-message delete option and give instructions to check the error details to fix.  I don't know of all the errors so I think I'm going to leave the full trace of the error to hopefully provide more context.
The one think I am removing is the initial `Error:` so it will be more like `Error invoking remote meth 'check-dataset':` followed by the context/info that caused that error.